packages/README.md

```diff
@@ -10,3 +10,11 @@ in compliance with [Flutter repo style guide]( https://github.com/flutter/flutte
 1. Public getter
 2. Private field
 3. Public setter (when needed)
+
+## Naming for methods and functions
+
+In compliance with [Flutter repo style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#naming-rules-for-typedefs-and-function-variables) we name:
+
+1. Typedefs: `FooCallback`
```

Comment:
I think this is different from what that link states.
The link says that these naming conventions are for callbacks and how we should be defining them as typedefs, with specific naming conventions.
i.e. 
```
typedef ExceptionCallback = void Function(dynamic exception);
class SomeWidget extends StatelessWidget {
  const SomeWidget({..., required this.onException });
  final ExceptionCallback onException;
  ...
}
Then when initializing the class
final handleException = (dynamic e) {
    exception = e;
};
SomeWidget(onException: handleException);
```

Reply:
Changed wording. Is it better?